src/app/services/seo.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Meta, Title } from '@angular/platform-browser';
+import { stringify } from 'querystring';
+
+@Injectable({
+  providedIn: 'root',
+})
+
+/*
+This service is used for SEO within the project (Search Engine Optimization). This means that
+all things related to modifying page metadata will be executed from this service.
+*/
+export class SEOService {
+
+  constructor(private title: Title, private meta: Meta) { }
+
+  // Updates the page title of the webpage
```

Comment:
Please use JSDoc style documentation: https://jsdoc.app/about-getting-started.html
Example: 
```ts
/**
   I am JSDoc documentation
*/
public someMethod() { }
```

Reply:
I was looking on the website and was wondering: What comment style should I be using in a small code snippet (explaining a single line of code)? 
For example, should I be using method A or B. The difference in these being the way the comment is opened/closed
A:
```
// Check x for reasons y and z
if (x){
    //do stuff//
}
```

B:
```
/** Check x for reasons y and z */
if (x) {
    /** do stuff  */
 }
```

Reply:
I would use option A for small code snippets. In general I use these as my guidelines:
- Method, class, interface, service -> Option B
- 1 or 2 lines of comment lines in the code -> Option A
- More than 2 lines of comments in the code -> Option B